# RNAseq tutorial demo at BUG
=========================================

- This is a tutorial for RNAseq from [guertin lab](http://guertinlab.org/wp-content/uploads/2015/02/ChIP_RNA_seq_analysis.pdf)

- The basic steps for the RNAseq workflow are as follows:

1. Download RNA-seq ENCODE data via UCSC
2. Align to the genome and splice junctions with Tophat
3. Count reads in genes with HTSeq
4. Assess differential expression with the DESeq package in R

The goal is to make into a git tutorial for BUG so that it can both be version controlled and also can be shared
